### PR TITLE
Set image mark down

### PR DESCRIPTION
**Pull request by cjm106**
This branch enables the images displayed, on both index and chapter pages, to be defined via markdown.

### CODE REVIEW

File: src/components/bookFrontCover/bookFrontCover.spec.tsx [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+          jobTitle={await getPageQueryData("index")}
+          careerAim={await getPageQueryData("index")}
+          image={await getPageQueryData("index")}
+        />
+      )
+
+      expect(wrapper).toHaveLength(1)
+    })
+  )
 })
```
Comment (Gina-Frankel) [inline src/components/bookFrontCover/bookFrontCover.spec.tsx:22]: 🥇

File: src/components/bookSection/bookSection.tsx [hunk 2]
```diff
@@ -15,19 +17,26 @@ interface Props {
...
+  let style, picture
+
   if (index % 2 === 0) {
     style = "book-mid-line"
   }
 
+  if (image != null) {
+    picture = <Img fluid={image.childImageSharp.fluid} />
+  }
+
```
Comment (Gina-Frankel) [inline src/components/bookSection/bookSection.tsx:31]: Could this be `if (image){...}...`

File: src/components/bookFrontCover/__snapshots__/bookFrontCover.spec.tsx.gql.json [hunk 3]
```diff
@@ -0,0 +1 @@
...pFRUWDvjkvngjrwSD1ridf112TSn6q2Enmbuey70XN3Mol6M9H33dKMjBwn3A6HZ4f7e3tvQzoREMjuFvINpKRH7KZd3oI9Ri6TqoOBldhrk1B2prK2F5CxYhz4+PjWjrPsMXy8nI1LzJw/d6nrUpg4JD2xoUE8uiKgtD8QauH0HzeOQjcZSrEkxYF6bl1dQFIRkZGSI26mjweHWVQFBUdPsIryQ1qwvBR0GTrms7GYag+E3hwhIJUeNr6EX5tyELV8Ujsi/dEaUE4xkYfET2RjomJCa6nu1s3NfVsdnJyErsyMqJ5aRHih5c/iwEtBFf7aQTpoJVt+zoF5wtCkSATIzFIglhfCXztrLA7IW4hXeT5/A0ODoL2pCNP5uJwOmWrJ70AtswlyVxJVrgX5DJPEuvthmDxRhzbFYAvTypRVlqKmtpLZPj3EaLRaNB64/p0Y2Pj8IXKyt6S4uJr2QrFcf19Gu3j9IkyJhDZO/ynFBH+XJrMm0sMlnCnFDLuemUi91tTFqfrzOU09QdIb30BKVVFaiOkjthgZlpHAaIXetDdzjqEXpagFyjSt3nieGYwutRy4P4h3LmcjrGrOcA92gF9tEgPCnHl7HskKVgK8Xqrz9l+Gq2QnRZCuL+Pb4DjenmIxF7dVBr/iDbuKAU81nXkjY1fyxmjp2acNvcE7ctJDHz8rPGL+CnL1avUIhHfnO01WWbMt1lrvcj6C4qMvzd6xmdMAAAAAElFTkSuQmCC","aspectRatio":0.8928571428571429,"src":"/static/1e0d5063347ac46ac94f97c07aac8a41/37d5a/open-peeps.png","srcSet":"/static/1e0d5063347ac46ac94f97c07aac8a41/5acf8/open-peeps.png 75w,\n/static/1e0d5063347ac46ac94f97c07aac8a41/95f17/open-peeps.png 150w,\n/static/1e0d5063347ac46ac94f97c07aac8a41/37d5a/open-peeps.png 300w,\n/static/1e0d5063347ac46ac94f97c07aac8a41/125bc/open-peeps.png 358w","sizes":"(max-width: 300px) 100vw, 300px"}}}},"html":""}}]}}}
```
Comment (Gina-Frankel) [inline src/components/bookFrontCover/__snapshots__/bookFrontCover.spec.tsx.gql.json:1]: Add newline at end of file

**Review by jamesAforster — APPROVED** [review]

File: src/components/developerImage/developerImage.tsx [hunk 4]
```diff
@@ -1,15 +1,19 @@
...
+
+export const DeveloperImage = (props: Props) => {
+  const { image } = props
   return (
     <div>
       <div className="row">
         <div className="col-6 offset-3 pt-5">
-          <Image />
+          <Img fluid={image.childImageSharp.fluid} />
         </div>
```
Comment (Gina-Frankel) [inline src/components/developerImage/developerImage.tsx:17]: Maybe we could deconstruct this further? 

See the paragraph **Extracting properties from nested objects**  in the article:  https://dmitripavlutin.com/javascript-object-destructuring/#:~:text=The%20object%20destructuring%20is%20a,the%20property%20doesn't%20exist.

File: src/components/bookSection/bookSection.tsx [hunk 5]
```diff
@@ -1,11 +1,13 @@
 import React from "react"
 import "bootstrap/dist/css/bootstrap.css"
 import "./bookSection.scss"
+import Img from "gatsby-image"
 
 interface BodyItem {
   title: string
   subTitle: string
   content: string
+  image: any
```
Comment (Gina-Frankel) [inline src/components/bookSection/bookSection.tsx:10]: Is there any more specific we can use? Or is any what is required for an image?

**Review by Gina-Frankel — CHANGES_REQUESTED** [review]
I think before we merge we should put all the images for the DoublePage component - just so then it is like a complete task 
I would be more than happy to do that, as it would be great for me to practice sending images on graphql :)

File: src/components/bookFrontCover/__snapshots__/bookFrontCover.spec.tsx.gql.json (hunk 3 above)
Comment (cjm106) [reply src/components/bookFrontCover/__snapshots__/bookFrontCover.spec.tsx.gql.json:1]: This file is generated by the gatsby-test-plugin, I'd prefer to leave it as is.

File: src/components/developerImage/developerImage.tsx (hunk 4 above)
Comment (cjm106) [reply src/components/developerImage/developerImage.tsx:17]: Its possible, the above is based on Gatsby documentaion  for Img/fluid - https://www.gatsbyjs.com/plugins/gatsby-image/

File: src/components/bookSection/bookSection.tsx (hunk 2 above)
Comment (cjm106) [reply src/components/bookSection/bookSection.tsx:31]: That wasn't working for me at the time i implemented the above.  I've changed it back and oddly its now working - the error must've been caused by something else..

File: src/components/bookSection/bookSection.tsx (hunk 5 above)
Comment (cjm106) [reply src/components/bookSection/bookSection.tsx:10]: Any is the best option at this stage.  Anything else results in all tests failing within the CI environment -  the following error message is the reason I changed it to any 'TS2339: Property 'childImageSharp' does not exist on type 'object'.'

File: src/components/developerImage/developerImage.tsx (hunk 4 above)
Comment (cjm106) [reply src/components/developerImage/developerImage.tsx:17]: dencostructing results in the following error - 
TypeError: Cannot read property 'childImageSharp' of null

Comment (cjm106) [reply src/components/developerImage/developerImage.tsx:17]: It doesnt appear possible to deconstruct this any further.  Suggest we keep as is for the time being

**Pull request merged**